app.py

```diff
@@ -1,17 +1,23 @@
 #!/usr/bin/env python3
 import os
 import boto3 
+import string
+import random
 import aws_cdk as cdk
 import aws_cdk.assertions as assertions
 from sds_in_a_box.sds_in_a_box_stack import SdsInABoxStack
 
-
+### Implement the code found in "sds_in_a_box_stack.py"
 app = cdk.App()
-SdsInABoxStack(app, "SdsInABoxStack")
 
-app.synth()
+SDS_ID = app.node.try_get_context("SDSID")
+
+if SDS_ID is None:
+    raise Exception("ERROR: Need to specify an ID to name the stack (ex - production, testing, etc)")
+elif SDS_ID=="random":
+    # A random unqiue ID for this particular instance of the SDS
+    SDS_ID = "".join( [random.choice(string.ascii_lowercase) for i in range(8)] )
+    
 
-#app = cdk.App()
-#stack = SdsInABoxStack(app, "sds-in-a-box")
-#template = assertions.Template.from_stack(stack)
-#print(template)
\ No newline at end of file
+SdsInABoxStack(app, "SdsInABoxStack"+SDS_ID)
```

Comment:
Can we add a space or `-` between `SdsInABoxStack` and `SDS_ID`? When I was using random string, it becomes hard to read what is SDS_ID.